docker/readme.md

```diff
@@ -0,0 +1,54 @@
+Docker Script Documentation
+===========================
+
+The files in this folder are intended to be mainly used by PyNE maintainers,
+they provide an "easy" mechanism to build and push the various containers
+required for the multiple configurations tested by the PyNE CI.
+
+This folder contains:
+
+- A custom Dockerfile: "ubuntu_18.04-dev.dockerfile", allowing the user 
+to build docker containers with a custom set of
+  dependencies required by the CI. The configuration of the Docker containers
+  can be specified using the `--build-args <flag>` argument. The different flags are:
+    
+    - `build_moab=YES` add MOAB in the docker container
```

Comment:
Are these the default values?  Presumably you can say either `YES` or `NO` for each option?